css-two-column-layout/styles.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Karla', sans-serif;
+}
+
+.container {
+  align-items: center;
+  margin: auto;
+  justify-content: center;
+  width: 900px;
+}
+
+.row {
+  display: flex;
+}
+
+.column-half {
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+}
+
+.column-one-third {
+  display: flex;
+  flex-direction: column;
+  width: calc(100% / 3);
+}
+
+.move-one {
+  margin-right: 70px;
+}
+
+.move-two {
```

Comment:
These class names don't really explain their purpose. I recommend `margin-right-70` and `margin-left-70`